packages/site-kit/src/lib/styles/base.css

```diff
@@ -45,7 +45,7 @@ p,
 ul,
 ol {
 	font: var(--sk-font-body);
-	margin: 1lh 0;
+	margin: 0.5lh 0;
```

Comment:
I agree that `1lh` seems like a lot and we could do less. it does feel a little close together now, but not by much. what do you think about giving it just a tad more at `0.6lh`?
```suggestion
	margin: 0.6lh 0;
```

Reply:
Typography people like to talk about 'vertical rhythm' — that's where the `1lh` came from in the first place. We're breaking that rule in a lot of places (we'd need to build the whole thing from scratch rather than merely changing the old site if we wanted to be really serious about it) but I do think that 'things should be multiples of each other' is sage advice that we should adhere to:

<img width="1767" alt="image" src="https://github.com/user-attachments/assets/883a52fa-0af0-4d7d-a1fe-f0cc8b760af6">


With `0.6lh`, everything becomes... bumpy. Just very slightly off:

<img width="1734" alt="image" src="https://github.com/user-attachments/assets/e5988f88-75bc-4b67-87fa-6cdcf59a0424">